src/amo/components/App/index.js

```diff
@@ -26,7 +26,7 @@ import { getDjangoBase62 } from 'amo/utils';
 import { logOutUser as logOutUserAction } from 'amo/reducers/users';
 import { addChangeListeners } from 'core/addonManager';
 import { setUserAgent as setUserAgentAction } from 'core/actions';
-import { setInstallState } from 'core/actions/installations';
+import { setInstallState } from 'core/reducers/installations';
```

Comment:
so... this doesn't change any functionality, just move the code around?  (If so, couldn't all of the remaining files in /actions/ have been done at the same time?)

Reply:
Yes, it only moves code. This is needed for another patch so I'd like to keep this patch small and focused. Other action files might have more impact on the codebase.. (because legacy).

Reply:
I'm not sure why it wasn't just a second commit included with the other patch then, but 🤷

Reply:
The idea is to reduce the overall size and complexity of the next patch. This is not always possible when a commit cannot really be merged alone but in this case, this is a cleanup we wanted to do anyway so it's fine to land it alone. Even if the next patch does not get merged for some reason, this particular patch is still useful and self-contained.